package.json

```diff
@@ -9,8 +9,9 @@
     "link": "cd build && npm link",
     "link:vite-project": "npm link",
     "publish:build": "npm run build && cd build && npm publish",
-    "eslint": "eslint --max-warnings 0 src/",
-    "es-check": "npm run build && es-check es5 './build/**/*.js'",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix .",
+    "prettier": "prettier --write .",
```

Comment:
`"prettier": "prettier --write .",` this shouldn't be necessary

Reply:
removed it